### DISCUSSION TITLE
triggeredAlarmState The interface fails to obtain alarm data

### DISCUSSION THREAD

**Discussion by iidst** [vmware/pyvmomi#1067] [Q&A] [answered]
### Describe the problem

I see three alarms of esxi hosts on vcenter
![1](https://github.com/vmware/pyvmomi/assets/90675896/212de73b-c312-4712-ad20-ea9816802829)
However, I use the triggeredAlarmstate interface of host-system to get only two, how do I get the third one, or use other interfaces
![3](https://github.com/vmware/pyvmomi/assets/90675896/c9e480b1-1108-4bac-8cc8-f2ac8f05767c)

### Code

```
content = si.RetrieveContent()
vc = content.rootFolder.childEntity
for datacenter in vc:
    for databases in datacenter.hostFolder.childEntity:
        for exsi in databases.host:
            print('-->' + exsi.name)
            for exsitriggered in exsi.triggeredAlarmState:
                print(exsitriggered)
```

Thanks...

**Comment by DanielDraganov** [comment, marked as answer]
Hello,
This is not a pyVmomi issue but a misconfiguration or hardware/setup issue.
Moving this to Discussions.
